Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView/Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView.csproj

```diff
@@ -0,0 +1,107 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{81B7758C-233E-489A-8F6D-0EC25660F46E}</ProjectGuid>
+    <OutputType>WinExe</OutputType>
+    <RootNamespace>Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView</RootNamespace>
```

Comment:
@nmetulev Feedback on the name. Any ideas?

Reply:
We should follow the existing naming convention here. I'd expect two sample projects grouped by framework: 'Microsoft.Toolkit.Win32.WPF.SampleApp' and 'Microsoft.Toolkit.Win32.WinForms.SampleApp'

Reply:
I'll rename the other two, but this one is a third (there should also be a fourth that shows the WPF control working in WinForms). What should the latter two be named?

Reply:
Also, how does this reconcile against the Background tasks sample? Am I renaming the other two, or building a generic sample app like the UWP one?

Reply:
Can we have only two generic samples instead of creating multiple sample projects? For example, the WPF sample app would host all WPF samples?

The background tasks sample is needed for the UWP SampleApp (that sample is demonstrating background task)

Reply:
@nmetulev How complex should this be? The UWP version is quite. I'm not willing to hold up a PR for not having a generic WinForms and WPF sample platform, rather I would omit these samples and the generic sample host on YAGNI and wait until we have a need for them in the upcoming months